tools/run_all_tests.sh

```diff
@@ -68,24 +68,53 @@ echo ">>>>>> Running Ice0ryx Tests <<<<<<"
 
 set -e
 
+execute_test () {
+    local component=$1
+    local test_scope=$2
+
+    case $test_scope in
+        "unit")
+        test_binary="$component"_moduletests
```

Comment:
Variables should be `ALL_CAPS` according to the existing convention in this file.